# This message sends relevant vectornav info to velocity prediction model

std_msgs/Header header
geometry_msgs/Pose2D linear
float64 angular_velocity
